Description: 
 The original CMakeFile.txt used a different name for the static library.
 This difference was redundant with the file extensions and harmful for
 automatic lookup by library name.
Author: Simon Chopin <chopin.simon@gmail.com>
Forwarded: no
Last-Update: 2011-11-28
--- a/fplib/CMakeLists.txt
+++ b/fplib/CMakeLists.txt
@@ -30,7 +30,7 @@
 ADD_LIBRARY(lastfmfp_static STATIC ${LASTFM_FP_SOURCES})
 ADD_LIBRARY(lastfmfp_shared SHARED ${LASTFM_FP_SOURCES})
 
-SET_TARGET_PROPERTIES( lastfmfp_static PROPERTIES OUTPUT_NAME lastfmfp_static)
+SET_TARGET_PROPERTIES( lastfmfp_static PROPERTIES OUTPUT_NAME lastfmfp)
 SET_TARGET_PROPERTIES( lastfmfp_shared PROPERTIES OUTPUT_NAME lastfmfp)
 SET_TARGET_PROPERTIES( lastfmfp_shared PROPERTIES VERSION ${LASTFM_FP_MAJOR}.${LASTFM_FP_MINOR}.${LASTFM_FP_PATCH} SOVERSION ${LASTFM_FP_MAJOR})
 
